hw1/counter.js

```diff
@@ -0,0 +1,42 @@
+function createCounter(initialValue) {
+    const userVariable = {
+        userNumber: initialValue,
+        changeNumber: initialValue,
+    };
+
+    let {userNumber = 0, changeNumber = 0} = userVariable;
+
+    return {
+      increment(value) {
+        const incrementVariable = {
+            incrementNumber: value,
+        };
+
+        let {incrementNumber = 1} = incrementVariable;
+        changeNumber += incrementNumber;
```

Comment:
Couldn't you just use:
```suggestion
        changeNumber += value || 1;
``` 
?